config_src/drivers/nuopc_cap/mom_ocean_model_nuopc.F90

```diff
@@ -388,13 +388,14 @@ subroutine ocean_model_init(Ocean_sfc, OS, Time_init, Time_in, gas_fields_ocn, i
 
   call get_param(param_file, mdl, "USE_WAVES", OS%Use_Waves, &
        "If true, enables surface wave modules.", default=.false.)
+  ! MOM_wave_interface_init is called regardless of the value of USE_WAVES because
+  ! it also initializes statistical waves.
+  call MOM_wave_interface_init(OS%Time, OS%grid, OS%GV, OS%US, param_file, OS%Waves, OS%diag)
   if (OS%use_waves) then
-    call MOM_wave_interface_init(OS%Time, OS%grid, OS%GV, OS%US, param_file, OS%Waves, OS%diag)
-    call get_param(param_file,mdl,"SURFBAND_WAVENUMBERS",OS%Waves%WaveNum_Cen, &
-           "Central wavenumbers for surface Stokes drift bands.",units='rad/m', &
-           default=0.12566)
-  else
-    call MOM_wave_interface_init_lite(param_file)
+    ! I do not know why this is being set here.  It seems out of place. -RWH
+    call get_param(param_file,mdl,"SURFBAND_WAVENUMBERS", OS%Waves%WaveNum_Cen, &
+           "Central wavenumbers for surface Stokes drift bands.", &
+           units='rad/m', default=0.12566, scale=OS%US%Z_to_m)
```

Comment:
In my prototype implementation of the coupling with FMS I have these passed through the coupler, and not read from the input file (to ensure consistency with what is inside of a wave model).  Hence why in the coupler option of MOM_wave_interface (L311 of the code before these modifications) we only read the number of bands and allocate arrays in the "coupler" option, but don't read this field.  Presumably this is a temporary work around for nuopc to avoid breaking my codes until these features are more concretely established on our end, though @JessicaMeixner-NOAA may elaborate further if there are other reasons.

Reply:
To clarify, "L311 of the code" refers the mom_wave_interface code, not this particular file.

Reply:
I don't know of a current way to be able to pass this information from the coupler with NUOPC like it's done withe FMS.  So the way I was getting these variables to the ocean model was to just set them and read them in from the input file.  It's not ideal in the sense that it does allow for human error for the wave and ocean model to have different values here.

Reply:
This is a good explanation for why we need to leave this call here, at least for now. @JessicaMeixner-NOAA, do you think that it would it be OK if the comment is left as a reminder to revisit this some day, or should my commenting questioning why SURFBAND_WAVENUMBERS is set here be removed?

Reply:
For now, we are going to leave this as-is with the comment noting the unusual position for the call to set  SURFBAND_WAVENUMBERS.  @JessicaMeixner-NOAA, if you want to follow this up with a PR replacing this comment with an explanation, or if you want to move this call from the NUOPC coupler to mimic the other couplers, that would be welcomed.

Reply:
@Hallberg-NOAA sounds good